<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>authentification</class>
 <widget class="QDialog" name="authentification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>861</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLineEdit" name="password_labelc">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>340</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #dcdcdc;
  border-radius: 4px;
  box-sizing: border-box;
  background: #FFD700;
  color: #FFFFFF;
  font-size: 20px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_labelc">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>270</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #dcdcdc;
  border-radius: 4px;
  box-sizing: border-box;
  background: #FFD700;
  color: #FFFFFF;
  font-size: 20px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="duree_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #808080;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="text">
      <string>Prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #A9A9A9;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="text">
      <string>Veuillez remplir les informations suivantes pour avoir un Compte</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_labelc">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #dcdcdc;
  border-radius: 4px;
  box-sizing: border-box;
  background: #FFD700;
  color: #FFFFFF;
  font-size: 20px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_creer_compte">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>430</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:20px;
font-family:Arial, Helvetica, sans-serif;
border: 1px solid #DCDCDC;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background:#FFD700;
border: none;
}
</string>
     </property>
     <property name="text">
      <string>Cr√©er </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="prenom_labelc">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #dcdcdc;
  border-radius: 4px;
  box-sizing: border-box;
  background: #FFD700;
  color: #FFFFFF;
  font-size: 20px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="id_film_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #808080;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
    <widget class="QLabel" name="note_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #808080;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="note_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #808080;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nomsalle_labelc">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #dcdcdc;
  border-radius: 4px;
  box-sizing: border-box;
  background: #FFD700;
  color: #FFFFFF;
  font-size: 20px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="nom_film_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #808080;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QLineEdit" name="nv_prenomC">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>210</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #dcdcdc;
  border-radius: 4px;
  box-sizing: border-box;
  background: #FFD700;
  color: #FFFFFF;
  font-size: 20px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="nv_passwordC">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>340</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #dcdcdc;
  border-radius: 4px;
  box-sizing: border-box;
  background: #FFD700;
  color: #FFFFFF;
  font-size: 20px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="nv_nomC">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #dcdcdc;
  border-radius: 4px;
  box-sizing: border-box;
  background: #FFD700;
  color: #FFFFFF;
  font-size: 20px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_modC">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>430</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:20px;
font-family:Arial, Helvetica, sans-serif;
border: 1px solid #DCDCDC;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background:#FFD700;
border: none;
}
</string>
     </property>
     <property name="text">
      <string>Cr√©er </string>
     </property>
    </widget>
    <widget class="QLabel" name="note_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #808080;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nv_usernameC">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>270</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #dcdcdc;
  border-radius: 4px;
  box-sizing: border-box;
  background: #FFD700;
  color: #FFFFFF;
  font-size: 20px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="note_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #808080;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="duree_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #808080;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="text">
      <string>Prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>611</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #A9A9A9;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="text">
      <string>Veuillez remplir les informations suivantes pour modifier un Compte</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_ex_c">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #dcdcdc;
  border-radius: 4px;
  box-sizing: border-box;
  background: #FFD700;
  color: #FFFFFF;
  font-size: 20px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="nom_film_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #808080;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="id_film_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #808080;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Supprimer</string>
    </attribute>
    <widget class="QLineEdit" name="c_a_supp">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_supprimerC">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>370</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Id du compte</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
    <widget class="QTableView" name="tabCompte">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>451</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_refreshC">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>370</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pdfC">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_searchC">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>430</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="compte_a_chercher">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>430</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Compte √† chercher</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
